timed/employment/filters.py

```diff
@@ -37,6 +37,9 @@ class Meta:
 
 
 class UserFilterSet(FilterSet):
+    active = NumberFilter(name='is_active')
+    supervisor = Filter(name='supervisors__id', lookup_expr='contains')
```

Comment:
This is a fairly fancy filter I think it would be good to add a test (best in pytest style)

Reply:
Done!